cocos/asset/assets/asset.jsb.ts

```diff
@@ -48,21 +48,22 @@ assetProto._ctor = function () {
 
     this._callbackTable = js.createMap(true);
     this._file = null;
+    this._uuid = ''; // @deprecated since v3.5.0
```

Comment:
`_uuid` is a jsb property. no need to initialize it here.
```c++
bool js_register_cc_Asset(se::Object* obj) {
    auto* cls = se::Class::create("Asset", obj, __jsb_cc_CCObject_proto, _SE(js_new_cc_Asset)); 
    
    cls->defineStaticProperty("__isJSB", se::Value(true), se::PropertyAttribute::READ_ONLY | se::PropertyAttribute::DONT_ENUM | se::PropertyAttribute::DONT_DELETE);
    cls->defineProperty("_native", _SE(js_cc_Asset__native_get), _SE(js_cc_Asset__native_set)); 
    cls->defineProperty("_nativeUrl", _SE(js_cc_Asset__nativeUrl_get), _SE(js_cc_Asset__nativeUrl_set)); 
    cls->defineProperty("_uuid", _SE(js_cc_Asset__uuid_get), _SE(js_cc_Asset__uuid_set)); 
//...
```

Reply:
fixed